src/coreclr/jit/targetamd64.h

```diff
@@ -78,10 +78,16 @@
 #endif // !UNIX_AMD64_ABI
   #define CSE_CONSTS               1       // Enable if we want to CSE constants
 
-  #define RBM_ALLFLOAT            (RBM_XMM0 | RBM_XMM1 | RBM_XMM2 | RBM_XMM3 | RBM_XMM4 | RBM_XMM5 | RBM_XMM6 | RBM_XMM7 | RBM_XMM8 | RBM_XMM9 | RBM_XMM10 | RBM_XMM11 | RBM_XMM12 | RBM_XMM13 | RBM_XMM14 | RBM_XMM15)
+  #define RBM_LOWFLOAT            (RBM_XMM0 | RBM_XMM1 | RBM_XMM2 | RBM_XMM3 | RBM_XMM4 | RBM_XMM5 | RBM_XMM6 | RBM_XMM7 | RBM_XMM8 | RBM_XMM9 | RBM_XMM10 | RBM_XMM11 | RBM_XMM12 | RBM_XMM13 | RBM_XMM14 | RBM_XMM15 )
+  #define RBM_HIGHFLOAT           (RBM_XMM16 | RBM_XMM17 | RBM_XMM18 | RBM_XMM19 | RBM_XMM20 | RBM_XMM21 | RBM_XMM22 | RBM_XMM23 | RBM_XMM24 | RBM_XMM25 | RBM_XMM26 | RBM_XMM27 | RBM_XMM28 | RBM_XMM29 | RBM_XMM30 | RBM_XMM31)
```

Comment:
I wonder if we could come up with a better name than "HIGHFLOAT"

From the initial `8` SIMD registers, we've had 2 extensions so far and its conceivable a 3rd could happen in the future

Reply:
I've used "high" to refer to the 16 AVX512 registers mostly because that is the term reference in the Intel manual, with "next" being used to reference the 8 AVX registers.

I'm in favor of a better naming scheme, but not certain what to choose. We could select `RBM_AVX512FLOAT` or something along those lines...

Reply:
I don't have a particular preference and I don't think it's "strictly" necessary to update. More a passing thought about how it might be confusing.